docs/developers.html

```diff
@@ -0,0 +1,209 @@
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store User Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Disaster Recovery Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Gluster Hyperconvergence Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Orb</a></li>
+        <li class="ov-nav-main__sub-item ov-nav-main__sub-item--heading">Release notes</li>
+        <li class="ov-nav-main__sub-item">Latest Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item">Older Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="developers.html">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="community.html">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="https://lists.ovirt.org/archives/">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
+  </ul>
+</nav>
+  </header>
+<br>
+<br>
+   <section class="ov-site-section ov-masthead--community">
+      <h2 class="ov-heading ov-heading--lg">Developer Documentation</h2>
+      <br>
+      <br>
+      <p>(add warning icon) Much of this developer documentation is outdated, but provides historical context. It is not user documentation and should not be treated as such. Documentation is available here. 
+      </p>
+   </section>
+
+    <section class="ov-site-section ov-masthead--community">
+      <h2 class="ov-heading ov-heading--lg">Design Documentation</h2>
+      <br>
+      <div>
+        <br>
+        <ul style="list-style-type:disc;">
+          <a href="#"><li>Feature Pages / Design Documentation</li></a>
+          <a href="#"><li>Architecture (outdated)</li></a>
+        </ul> 
+      </div>
+    </section>
+
+    <section class="ov-site-section ov-masthead--community">
+      <h2 class="ov-heading ov-heading--lg">Containerization</h2>
+      <br>
+      <div>
+        <br>
+        <ul style="list-style-type:disc;">
```

Comment:
Another good case for a general content wrapper so we can style the lists the way you want them.

Reply:
So in this case would we create a specific class for the wrapper in CSS and then assign it to all of the lists so they are consistent?